onnx/cpp2py_export.cc

```diff
@@ -64,19 +64,27 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
         return py::bytes(bytes);
       })
       .def_property_readonly("has_context_dependent_function", &OpSchema::HasContextDependentFunction)
-      .def("get_context_dependent_function", [](OpSchema* op, const py::bytes& bytes) -> py::bytes {
-        NodeProto proto{};
-        ParseProtoFromPyBytes(&proto, bytes);
-
-        std::string func_bytes = "";
-        if (op->HasContextDependentFunction()) {
-          FunctionBodyBuildContextImpl ctx(proto);
-          FunctionProto func_proto;
-          op->BuildContextDependentFunction(ctx, func_proto);
-          func_proto.SerializeToString(&func_bytes);
-        }
-        return py::bytes(func_bytes);
-      });
+      .def(
+          "get_context_dependent_function",
```

Comment:
I feel it's better to have a test for this function.

Reply:
Agree with test-case. But I don't think we can do it from python since we need to define a new dummy function for this purpose, which needs to be done in C++. Let me look into this separately because some test-infrastructure is required I think.